tex/latex/biblatex/lbx/latvian.lbx

```diff
@@ -169,152 +170,242 @@
+                      {izdev\adddot, anot\adddotspace un p\={e}cv\adddotspace autors}},
+  editorsanaf      = {{izdev\={e}ji, anot\={a}cijas un p\={e}cv\={a}rda autori}%
+                      {izdev\adddot, anot\adddotspace un p\={e}cv\adddotspace autori}},
+  editortrcoin     = {{izdev\={e}js, tulkot\={a}js, koment\={e}t\={a}js un ievada autors}%
+                      {izdev\adddot, tulk\adddot, koment\adddotspace un ievada autors}},
+  editorstrcoin    = {{izdev\={e}ji, tulkot\={a}ji, koment\={e}t\={a}ji un ievada autori}%
+                      {izdev\adddot, tulk\adddot, koment\adddotspace un ievada autori}},
+  editortrcofo     = {{izdev\={e}js, tulkot\={a}js, koment\={e}t\={a}js un priek\v{s}v\={a}rda autors}%
+                      {izdev\adddot, tulk\adddot, koment\adddotspace un priek\v{s}v\adddotspace autors}},
+  editorstrcofo    = {{izdev\={e}ji, tulkot\={a}ji, koment\={e}t\={a}ji un priek\v{s}v\={a}rda autori}%
+                      {izdev\adddot, tulk\adddot, koment\adddotspace un priek\v{s}v\adddotspace autori}},
+  editortrcoaf     = {{izdev\={e}js, tulkot\={a}js, koment\={e}t\={a}js un p\={e}cv\={a}rda autors}%
+                      {izdev\adddot, tulk\adddot, koment\adddotspace un p\={e}cv\adddotspace autors}},
+  editorstrcoaf    = {{izdev\={e}ji, tulkot\={a}ji, koment\={e}t\={a}ji un p\={e}cv\={a}rda autori}%
+                      {izdev\adddot, tulk\adddot, koment\adddotspace un p\={e}cv\adddotspace autori}},
+  editortranin     = {{izdev\={e}js, koment\={e}t\={a}js, anot\={a}cijas un ievada autors}%
+                      {izdev\adddot, koment\adddot, anot\adddotspace un ievada autors}},
+  editorstranin    = {{izdev\={e}ji, koment\={e}t\={a}ji, anot\={a}cijas un ievada autori}%
+                      {izdev\adddot, koment\adddot, anot\adddotspace un ievada autori}},
+  editortranfo     = {{izdev\={e}js, koment\={e}t\={a}js, anot\={a}cijas un priek\v{s}v\={a}rda autors}%
+                      {izdev\adddot, koment\adddot, anot\adddotspace un priek\v{s}v\adddotspace autors}},
+  editorstranfo    = {{izdev\={e}ji, koment\={e}t\={a}ji, anot\={a}cijas un priek\v{s}v\={a}rda autori}%
+                      {izdev\adddot, koment\adddot, anot\adddotspace un priek\v{s}v\adddotspace autori}},
+  editortranaf     = {{izdev\={e}js, koment\={e}t\={a}js, anot\={a}cijas un p\={e}cv\={a}rda autors}%
+                      {izdev\adddot, koment\adddot, anot\adddotspace un p\={e}cv\adddotspace autors}},
+  editorstranaf    = {{izdev\={e}ji, koment\={e}t\={a}ji, anot\={a}cijas un p\={e}cv\={a}rda autori}%
+                      {izdev\adddot, koment\adddot, anot\adddotspace un p\={e}cv\adddotspace autori}},
+  translatorco     = {{tulkot\={a}js un koment\={e}t\={a}js}{tulk\adddotspace un koment\adddot}},
+  translatorsco    = {{tulkot\={a}ji un koment\={e}t\={a}ji}{tulk\adddotspace un koment\adddot}},
+  translatoran     = {{tulkot\={a}js un anot\={a}cijas autors}%
+                      {tulk\adddotspace un anot\adddot autors}},
```

Comment:
There would not be a space after the dot here in 'anot.autors'. Is that intended?

Reply:
There should be, fixed